src/staking-router/staking-router.service.ts

```diff
@@ -13,6 +15,77 @@ export class StakingRouterService {
     protected readonly keysApiService: KeysApiService,
   ) {}
 
+  public async getVettedAndUnusedKeys() {
+    // TODO: add cache by modules nonce
+    const operatorsByModules =
+      await this.keysApiService.getOperatorListWithModule();
+    const operatorsBlockHash =
+      operatorsByModules.meta.elBlockSnapshot.blockHash;
+    const operatorsBlockNumber =
+      operatorsByModules.meta.elBlockSnapshot.blockNumber;
+
+    const unusedKeys = await this.keysApiService.getUnusedKeys();
```

Comment:
If we decided to load all data at once, we need to calculate at what point in time memory will be insufficient and what to do in this situation

Reply:
blockHash of two requests is checking down as it was before in getStakingModuleUnusedKeys method

Maybe we could consider here use of /v2/modules/operators/keys endpoint, but than we need to add 'used' filter
because at the moment we could fetch only all keys

Reply:
added task for it https://linear.app/lidofi/issue/VAL-493/solving-problem-with-vetted-keys-fetching